components/match2/wikis/ageofempires/match_group_input_custom.lua

```diff
@@ -85,7 +85,8 @@ function CustomMatchGroupInput._getMapsAndGame(match)
 	Variables.varDefine('tournament_maps', data.maps)
 
 	match.game = match.game or data.game
-	match.mapsInfo = Logic.emptyOr(match.mapsInfo, Json.parse(data.maps))
+	local mapsInfo = Json.parse(data.maps)
```

Comment:
doesn't this just cause additional json parse calls?

Reply:
Possible, the reason is to prevent the second return parameter from being used in the emptyOr

Reply:
in that case just add `()` imo